core/src/display_object/edit_text.rs

```diff
@@ -1129,6 +1132,42 @@ impl<'gc> EditText<'gc> {
             }
         }
     }
+
+    fn initialize_as_broadcaster(&self, activation: &mut Activation<'_, 'gc, '_>) {
+        let write = self.0.write(activation.context.gc_context);
+        let object = write.object.unwrap();
+        activation.context.avm1.broadcaster_functions().initialize(
+            activation.context.gc_context,
+            object,
+            activation.context.avm1.prototypes().array,
+        );
+
+        if let Value::Object(listeners) = object.get("_listeners", activation).unwrap() {
```

Comment:
If we're matching here, might as well match on the full thing to avoid an unwrap:
`if let Ok(Value::Object(listeners)) = ...`